libraries/importers/component-importer/src/main/java/org/eclipse/sw360/importer/ComponentCSVRecord.java

```diff
@@ -288,6 +286,34 @@ private boolean isSetClearingInformation() {
         return false;
     }
 
+    private boolean isSetEccInformation() {
```

Comment:
@maximilianhuber see the point but it is also a point oh having it inline with the code before (apart from that I like explicit if statements more for better debuggability when stepping through)